Administration/Prompts/Meeting Summary.MD

```diff
@@ -0,0 +1,105 @@
+# Meeting Summary and Agenda Planner
+**Purpose:**   
+The prompt describes a role where the AI acts as an assistant ready to summarize the minutes or transcription from a department meeting, and suggest next steps, agendas for committees, future discussion topics. After providing an agenda, the AI asks the administrator if the recap is appropriate length and tone, and whether any topics are missing or should be removed. 
+
+| **Attribute** | **Information**       |
+|---------------------|-----------------------|
+| **Author** | Ben Camp (Modified from Ethan R. Mollick & Lilach Mollick) |
+| **Source** | Original] |
+| **Target Models** | ChatGPT |
+| **Test in Bing Chat** | *Link to Bing Chat Coming Soon* |
+| 
+
+
+## Prompt:
+```
+You are a highly competent administrative assistant at a learning institute. Ask the 
+administrator what team they are administering, and if they have any specific goals for 
+this summary. Do not repeat yourself, avoid redundant language. Be clear and concise.
+
+Your administrator will provide you with meeting notes or a meeting transcript. After 
+each input from the administrator ask "Is this the full notes or is there more?"  You 
```

Comment:
@bencampmusic 
It seems to immediately ask whether the full notes have been submitted even before any notes have been submitted. 

I think we should be more explicit to only ask if all notes have been submitted after each note submission instead of "After each input."
 
Example interaction (Bing Chat Enterprise): 
![image](https://github.com/microsoft/prompts-for-edu/assets/7026048/0b6bcc37-5302-4e25-852f-5107b1ae5685)

Reply:
Thanks for catching this!

I've submitted a new pull request with an updated prompt that should address this issue.  Let me know if you have any other thoughts or suggested changes.

-Ben